eng/Signing.props

```diff
@@ -5,114 +5,119 @@
+
+      <!-- Sign api-ms-win-core-xstate-l2-1-0 binary as it is only catalog signed in the current SDK. -->
+      <ItemsToSign
+        Condition="'$(Configuration)' == 'Release' and '$(TargetArchitecture)' == 'x86'"
+        Include="$(CoreCLRArtifactsPath)Redist\ucrt\DLLs\$(TargetArchitecture)\api-ms-win-core-xstate-l2-1-0.dll" />
+
+      <!-- Sign libraries. -->
+      <ItemsToSign Include="$(LibrariesNativeArtifactsPath)*.dll" />
+      <ItemsToSign Include="$(LibrariesSharedFrameworkRefArtifactsPath)*.dll" />
+      <!-- Most runtime artifacts will be crossgenned, so sign them post-crossgen. mscorlib isn't. -->
+      <ItemsToSign Include="$(LibrariesSharedFrameworkBinArtifactsPath)mscorlib.dll" />
+
+      <!-- Sign the host. -->
+      <ItemsToSign Include="$(BaseOutputRootPath)corehost/**/hostfxr.dll" />
+      <ItemsToSign Include="$(BaseOutputRootPath)corehost/**/hostpolicy.dll" />
+      <ItemsToSign Include="$(BaseOutputRootPath)corehost/**/dotnet.exe" />
+      <ItemsToSign Include="$(BaseOutputRootPath)corehost/**/ijwhost.dll" />
+      <ItemsToSign Include="$(BaseOutputRootPath)corehost/**/nethost.dll" />
+
+      <!-- Sign managed libraries in installer subset. -->
+      <ItemsToSign Include="$(ArtifactsBinDir)Microsoft.NET.HostModel/**/*.dll" />
+    </ItemGroup>
+
+    <!-- Sign ready-to-run binaries after crossgen is applied. -->
+    <ItemGroup Condition="'$(SignR2RBinaries)' == 'true'">
+      <ItemsToSign Include="$(CrossGenRootPath)**/*.dll" />
+    </ItemGroup>
+
+    <ItemGroup Condition="'$(SignMsiFiles)' == 'true'">
+      <ItemsToSign Include="$(ArtifactsPackagesDir)**/*.msi" />
+      <ItemsToSign Include="$(ArtifactsPackagesDir)**/*.cab" />
+    </ItemGroup>
+
+    <ItemGroup Condition="'$(SignBurnEngineFiles)' == 'true'">
+      <ItemsToSign Include="@(BundleInstallerEngineArtifact)" />
+    </ItemGroup>
+
+    <ItemGroup Condition="'$(SignBurnBundleFiles)' == 'true'">
+      <!-- Sign the bundles, now that the engine is reattached. Avoid re-signing the engine. -->
+      <ItemsToSign
+        Include="@(BundleInstallerExeArtifact)"
+        Exclude="@(BundleInstallerEngineArtifact)" />
+      <!-- Note: wixstdba is internal to the engine bundle and does not get signed. -->
+    </ItemGroup>
+
+    <ItemGroup Condition="'$(SignFinalPackages)' == 'true'">
+      <DownloadedSymbolPackages Include="$(DownloadDirectory)**\*.symbols.nupkg" />
+      <ItemsToSign Include="$(DownloadDirectory)**\*.nupkg" Exclude="@(DownloadedSymbolPackages)" />
+
+      <ItemsToSign Include="$(DownloadDirectory)**\*.deb" />
+      <ItemsToSign Include="$(DownloadDirectory)**\*.rpm" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <!-- External files -->
+      <ItemsToSign Remove="@(ItemsToSign->WithMetadataValue('Filename', 'Newtonsoft.Json'))" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <ItemsToSign Update="@(ItemsToSign)" Authenticode="$(CertificateId)" />
```

Comment:
Since this is now within a target, the `Update` attribute isn't needed.

Reply:
Can we leave this as is to keep it minimal? I expect to just revert the target once we consume the fix.

Reply:
I believe so, but @rainersigwald can confirm.